circuit_knitting/cutting/cut_finding/optimization_settings.py

```diff
@@ -38,16 +38,18 @@ class OptimizationSettings:
     If None is used as the random seed, then a seed is obtained using an
     operating-system call to achieve an unrepeatable randomized initialization.
 
-    NOTE: The current release only supports LO gate and wire cuts. LOCC
+    NOTE: The current release only supports LO gate and wire cuts. locc
```

Comment:
Why did you go lower case here out of curiosity?

Reply:
>Why did you go lower case here out of curiosity?

That was actually inadvertent, thank you for catching that. I went ahead and fixed that.